openedx/core/djangoapps/user_authn/views/registration_form.py

```diff
@@ -362,6 +362,7 @@ def __init__(self):
             "employment_status",
             "work_experience_level",
             "job_title",
+            "arabic_name",
```

Comment:
this field will be added in `error_message_dict` at line 203.
Please remove duplicate gender in that dict also.